### PR TITLE
python3Packages.snapcast: 2.1.2 -> 2.1.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 2.1.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120213` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.snapcast</li>
    <li>python39Packages.snapcast</li>
  </ul>
</details>

**Review by mweinelt — CHANGES_REQUESTED** [review]
Can we get some tests going? They are available when fetching from Git.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120213` at b98c417d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.snapcast</li>
    <li>python39Packages.snapcast</li>
  </ul>
</details>

**Comment by fabaff** [comment]
> Can we get some tests going? They are available when fetching from Git.

Yes, we can. The source was tagged with the wrong version number but besides that the tests run.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120213` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.snapcast</li>
    <li>python39Packages.snapcast</li>
  </ul>
</details>

File: pkgs/development/python-modules/snapcast/default.nix [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+  version = "2.1.3";
   disabled = !isPy3k;
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-ILBleqxEO7wTxAw/fvDW+4O4H4XWV5m5WWtaNeRBr4g=";
+  src = fetchFromGitHub {
+    owner = "happyleavesaoc";
+    repo = "python-snapcast";
+    # The current release has a wrong tag, missing period 2.13 instead 2.1.3
```
Comment (mweinelt) [inline pkgs/development/python-modules/snapcast/default.nix:17]: https://github.com/happyleavesaoc/python-snapcast/issues/39

Comment (fabaff) [reply pkgs/development/python-modules/snapcast/default.nix:17]: Thanks for opening the issue.

Comment (mweinelt) [reply pkgs/development/python-modules/snapcast/default.nix:17]: Fixed upstream, please update.

**Pull request merged**